### DISCUSSION TITLE
How do I get the creation time of the file mounted to gramine?

### DISCUSSION THREAD

**Discussion by llnut** [gramineproject/gramine#1107] [General]
I use the following configuration

```
{ path = "/mmap.txt", uri = "file:mmap.txt" }
```
mount `mmap.txt` to the "/" path of the gramine's file system, but when I use `stat/fstat/statx` to get the ctime/atime/mtime or btime of the file, the return value is always 0. But when I execute outside gramine, I can get the expected return value.

**Comment by dimakuv** [comment]
@jcsora This looks like a duplicate of https://github.com/gramineproject/gramine/issues/1108, right? If so, I'll go and delete this discussion, after your acknowledgement.
